File: apps/dert-api/src/dertinfo-models/DataTransferObject/DertOfDerts/DodSubmissionDto.cs
namespace DertInfo.Models.DataTransferObject.DertOfDerts
{
    public class DodSubmissionDto
    {
        public int Id { get; set; }
        public int GroupId { get; set; }
        public string GroupName { get; set; }
        public string GroupPictureUrl { get; set; }
        public string EmbedLink { get; set; }
        public string EmbedOrigin { get; set; }
        public string DertYearFrom { get; set; }
        public string DertVenueFrom { get; set; }
        public bool HasAnyResults { get; set; }
        public int NumberOfResults { get; set; }
        public bool IsPremier { get; set; }
        public bool IsChampionship { get; set; }
        public bool IsOpen { get; set; }
    }
}

File: apps/dert-api/src/dertinfo-models/Typewriter/DataTransferObject/DodSubmissionDto.ts

    export interface DodSubmissionDto {
        id: number;
        groupId: number;
        groupName: string;
        groupPictureUrl: string;
        embedLink: string;
        embedOrigin: string;
        dertYearFrom: string;
        dertVenueFrom: string;
        hasAnyResults: boolean;
        numberOfResults: number;
        isPremier: boolean;
        isChampionship: boolean;
        isOpen: boolean;
    }
